app/graphql/types/profile.rb

```diff
@@ -32,19 +37,55 @@ class Types::Profile < Types::BaseObject
     null: true,
     description: 'The character this profile has declared as their waifu or husbando'
 
-  field :waifu_or_husbando, String,
+  field :waifu_or_husbando, Types::WaifuOrHusbando,
     null: true,
-    description: 'The user-provided (unsanitized) string used to identify the role of the waifu'
+    description:
+      <<~DESCRIPTION.squish
+        The user-provided (unsanitized) string used to identify
+        the role of the waifu. 'Waifu' or 'Husbando'
+      DESCRIPTION
```

Comment:
This should probably be fixed to reflect that it's no longer a "user-provided (unsanitized) string", it's actually an enum

```suggestion
    description: "The properly-gendered term for the user's waifu"
```